src/Umbraco.Web.UI.Client/lib/tinymce/langs/en_US.js

```diff
@@ -0,0 +1,261 @@
+tinymce.addI18n('en_CA',{
```

Comment:
@RachBreeze I think this should be `en_US`.
We already have `en_CA` language here: https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/lib/tinymce/langs/en_CA.js

Reply:
Hi @bjarnef 

Oof yes :-) good spot thank you

Reply:
@RachBreeze did you gad any time to update the PR? 😎👍